### DISCUSSION TITLE
colab notebook demo fails

### DISCUSSION THREAD

**Discussion by LuciusMos** [camel-ai/camel#928] [Q&A] [answered]
On 09/10, I ran the colab notebook demo but it fails at `role_play_session = RolePlaying("Computer Programmer", "Gamer", task_prompt=task_prompt)`
[colab link] https://colab.research.google.com/drive/1AzP33O8rnMW__7ocWJhVBXjKziJXPtim?usp=sharing#scrollTo=-IhYiAWDme66
[version] v0.1.7.2
[error message]
Original task prompt:
Design a custom game using pygame

---------------------------------------------------------------------------
LocalProtocolError                        Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/httpx/_transports/default.py](https://localhost:8080/#) in map_httpcore_exceptions()
     71     try:
---> 72         yield
     73     except Exception as exc:

35 frames
LocalProtocolError: Illegal header value b'Bearer '

The above exception was the direct cause of the following exception:

LocalProtocolError                        Traceback (most recent call last)
LocalProtocolError: Illegal header value b'Bearer '

The above exception was the direct cause of the following exception:

APIConnectionError                        Traceback (most recent call last)
[/usr/local/lib/python3.10/dist-packages/openai/_base_client.py](https://localhost:8080/#) in _request(self, cast_to, options, remaining_retries, stream, stream_cls)
   1005 
   1006             log.debug("Raising connection error")
-> 1007             raise APIConnectionError(request=request) from err
   1008 
   1009         log.debug(

APIConnectionError: Connection error.

**Comment by Wendong-Fan** [comment, marked as answer]
Hey @LuciusMos , from the error message it's due to `APIConnectionError: Connection error` could you check your API or network status?
